infra/terraform/modules/data_backup/data.tf

```diff
@@ -0,0 +1,51 @@
+data "aws_region" "current" {
+  current = true
+}
+
+data "aws_iam_policy_document" "assume_role_policy" {
```

Comment:
Minor style point. I would probably prefer to have these IAM-related resources near the others using them (the `iam.tf` file).

It obviously doesn't make any difference from Terraform perspective - so don't worry about this.

Reply:
yep, fair enough - I've moved the IAM documents into `iam.tf`